test-executor/src/main/java/com/cognifide/aet/executor/RunnerCacheUpdater.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Automated Exploratory Tests
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.aet.executor;
```

Comment:
This may be a class copied from old codebase, but at first glance it looks like it is not needed.
The `update` method is called only right before `suiteStatusHandler.handle`, which access the same key.
Could we consult with @Skejven and/or @subiron if we could remove `RunnerCacheUpdater`? Or maybe the `update` method shuld be called in some other places?
(should have)